stable/prometheus-operator/hack/sync_prometheus_rules.py

```diff
@@ -54,6 +54,7 @@ def new_representer(dumper, data):
     'general.rules': ' .Values.defaultRules.rules.general',
     'k8s.rules': ' .Values.defaultRules.rules.k8s',
     'kube-apiserver.rules': ' .Values.kubeApiServer.enabled .Values.defaultRules.rules.kubeApiserver',
+    'kube-apiserver-error': ' .Values.kubeApiServer.enabled .Values.defaultRules.rules.kubeApiserverError',
```

Comment:
Shouldn't here be kube-apiserver-error.rules ?

Reply:
I think the name here should follow the corresponding rule file name:
kube-apiserver.rules.yaml -> kube-apiserver.rules
kube-apiserver-error.yaml -> kube-apiserver-error

So if we'd like to use 'kube-apiserver-error.rules', the rule file should be renamed to kube-apiserver-error.rules.yaml.

This is my understanding, though. Let me know if that's incorrect.